SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialWorldSettings.cpp

```diff
@@ -42,6 +40,11 @@ TSubclassOf<USpatialMultiWorkerSettings> ASpatialWorldSettings::GetMultiWorkerSe
 		// behaviour).
 		return USpatialMultiWorkerSettings::StaticClass();
 	}
+	else if (!IsMultiWorkerEnabled())
```

Comment:
Why was this moved around?

Reply:
Previously the mw override was enforced locally and in the cloud. Now the override is only for the editor we want the cloud build to be unaffected so I had to change the priority of the checks to ensure the local and cloud configs were written correctly.